src/test/java/attendance/AttendanceBookTest.java

```diff
@@ -0,0 +1,53 @@
+package attendance;
+
+import static org.assertj.core.api.AssertionsForClassTypes.*;
+
+import java.time.LocalDateTime;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import attendance.common.exception.AttendanceFileException;
+import attendance.domain.AttendanceFileReader;
+import attendance.domain.attendance.Attendance;
+import attendance.domain.attendance.AttendanceBook;
+
+public class AttendanceBookTest {
+    private AttendanceBook manager;
+    private static final String TEST_FILE = "/attendances.csv";
+
+    @BeforeEach
+    void setUp() throws AttendanceFileException {
+        var repository = AttendanceFileReader.from(TEST_FILE);
+        var lines = repository.getLines();
+        manager = AttendanceBook.from(lines);
+    }
```

Comment:
```suggestion
    private final AttendanceFileReader reader = AttendanceFileReader.from("/attendances.csv"); 
    private final AttendanceBook manager = AttendanceBook.from(reader.getLines());
```
1. junit5의 기본 라이프사이클은 메소드이기 때문에 beforeEach로 초기화할 필요는 없습니다.
https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle

2. repository가 아닌 reader아닌가요?

Reply:
이름을 수정하는 리펙터링 과정에서 실수가 있었네요..

repository라는 이름에 맞는 기능을 하는가? 다시 고민해보았습니다.

파일에 대한 CRUD에서도 단순히 R만 가졌으며, 도메인을 직접 생성하지 않았기에,
단순히 csv를 읽어오는 역할로 CsvReader로 네이밍을 수정했습니다.

도메인에 관한 부분이 없다고 생각하여, Utilliy에 배치하였는데 오리는 어떻게 생각하시나요?
오히려 도메인 생성의 책임을 주고, 도메인 객체로 관리하는 것이 더 좋았을까요?🤔

Reply:
네 말씀해주신 이유로 util로도 현재는 충분하다고 생각해요.